vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/java/com/vaadin/flow/data/renderer/LitRenderer.java

```diff
@@ -70,27 +70,13 @@ public class LitRenderer<SOURCE> extends Renderer<SOURCE> {
 
     private final String templateExpression;
 
-    private final String propertyNamespace;
-
     private final Map<String, ValueProvider<SOURCE, ?>> valueProviders = new HashMap<>();
     private final Map<String, SerializableBiConsumer<SOURCE, JsonArray>> clientCallables = new HashMap<>();
 
     private final String ALPHANUMERIC_REGEX = "^[a-zA-Z0-9]+$";
 
     private LitRenderer(String templateExpression) {
         this.templateExpression = templateExpression;
-
-        int litRendererCount = 0;
-        if (UI.getCurrent() != null) {
-            // Generate a unique (in scope of the UI) namespace for the renderer
-            // properties.
-            litRendererCount = UI.getCurrent().getElement()
-                    .getProperty("__litRendererCount", 0);
-            UI.getCurrent().getElement().setProperty("__litRendererCount",
-                    litRendererCount + 1);
-
-        }
-        propertyNamespace = "lr_" + litRendererCount + "_";
```

Comment:
Now that the format has changed, we could update the [example](https://github.com/vaadin/flow-components/blob/d4c616c89bdc25fb1366d8af771d6b3353082471/vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/lit-renderer.ts#L79) at `lit-renderer.ts`

Reply:
Thanks, done.